packages/@aws-cdk/aws-autoscaling/lib/volume.ts

```diff
@@ -66,6 +66,13 @@ export interface EbsDeviceOptionsBase {
    * @default {@link EbsDeviceVolumeType.GP2}
    */
   readonly volumeType?: EbsDeviceVolumeType;
+
+  /**
+   * The throughput that the volume supports, in MiB/s
```

Comment:
Can you add some docs here around min/max value? Also I think if you increase it above 750 you also
have to increase the iops. Maybe we should automatically set the iops based on throughput _if_ a
specific iops is not provided? Also add something that says this only applies to GP3

Reply:
Thank you for the comments @corymhall -- just seeing this now.

> Can you add some docs here around min/max value?

Happy to!

> Maybe we should automatically set the iops based on throughput if a
specific iops is not provided?

Do you know if cloudformation / the backing API will return a specific error if you turn it above 750 w/o an iops input or will the backing service just take a sane default? I can also poke at the system behavior and once I figure it out code it into the tests.

I will try to find the docs on this.

> Also add something that says this only applies to GP3

Sure thing!

Reply:
@corymhall - the requested changes have been added to the PR, please take a look when you get a chance.